README.md

```diff
@@ -1,2 +1,31 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+**Pseudocode for Fallout 4 hacking game**
+
+	guesses = 4 # Number of guesses used for loop
+	guesses_left = 4 # Used to track guesses left 
+	word = random word from database # The word to be checked agaisnt
+	word_length = number of characters in chosen word # Length of word
+
+	print 'The password is ' word_length ' characters long, you have ' guesses ' guesses to get it right!'
+
+	for i in range(0, guesses): # Loops as many times as guesses allowed or until broken by the loop
+		if i == 4 # If 4 guesses have been done print failure and break loop
```

Comment:
Are you following the Python convention that `range(0, N)` does not include `N`? If so then the loop will terminate before `i` reaches 4. Either way, it is better to be explicit as the reader may not be a Python programmer, e.g.
```
for i = 0, 1, ..., guesses-1
```